kof-operator/webapp/collector/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "collector",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "engines": {
+    "node": ">=18.18.2"
```

Comment:
Let's add to the kof-operator README the requirements about nodejs version installation. Otherwise make commands won't work.